--- configure.ac.orig	2010-01-27 16:57:48.000000000 +0100
+++ configure.ac	2011-01-21 20:16:57.161691068 +0100
@@ -86,7 +86,7 @@
 if test "$prefix" = "NONE"; then
 	theprefix="/usr/local"
 fi
-PACKAGE_PLUGINS_DIR=`eval echo $theprefix/lib/mediastreamer/plugins`
+PACKAGE_PLUGINS_DIR="${libdir}/mediastreamer/plugins"
 AC_DEFINE_UNQUOTED(PACKAGE_PLUGINS_DIR,  "$PACKAGE_PLUGINS_DIR" ,[path of plugins])
 AC_SUBST(PACKAGE_PLUGINS_DIR)
 
